Applied-Upstream: commit: 910064463237a51249bdb5baaf6a8a6a1cd68d04
Author: Julien Guillot <julienguillot@protonmail.com>
Last-Update: 2022-03-21
Description: [PATCH] Remove check for window  hints when removing button in Icon
 tasklist

These can be updated AFTER we added the button

Fixes #78
---
 src/applets/icon-tasklist/IconTasklistApplet.vala | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/applets/icon-tasklist/IconTasklistApplet.vala b/src/applets/icon-tasklist/IconTasklistApplet.vala
index 47df6505..2abfe707 100644
--- a/src/applets/icon-tasklist/IconTasklistApplet.vala
+++ b/src/applets/icon-tasklist/IconTasklistApplet.vala
@@ -399,10 +399,6 @@ public class IconTasklistApplet : Budgie.Applet {
 	}
 
 	private void on_app_closed(Budgie.Abomination.RunningApp app) {
-		if (app.get_window().is_skip_pager() || app.get_window().is_skip_tasklist()) { // window not managed in the first place
-			return;
-		}
-
 		IconButton? button = this.buttons.get(app.id.to_string());
 
 		if (button == null && app.app_info != null) { // Button might be pinned, try to get button from launcher instead
-- 
2.34.1

